games/applications/run_ft.py

```diff
@@ -0,0 +1,102 @@
+# ------------------------------------------------------------------------------
+# Part of Red Queen Project.  This file is distributed under the MIT License.
+# See accompanying file /LICENSE for details.
+# ------------------------------------------------------------------------------
+
+"""Benchmark for Quantum Fourier Transform"""
+
+import os
+import random
+import pytest
+from applications import backends, run_qiskit_circuit
+import numpy as np
+
+# Importing standard Qiskit libraries
+from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister
+
+DIRECTORY = "qasm\\qft\\"
+QASM_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), DIRECTORY)
+SECRET_INT = random.randint(1, 20)
```

Comment:
You are correct, it shouldn't be random each time. This would cause some unfair advantages in fidelity as some machines would run a larger benchmark than others. I will correct this in a future commit. Thank you for your helpful comment.